### PR TITLE
update slider, DEBUGV for trip algo, fix without charge control

### PR DESCRIPTION
**Pull request by jlayec**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -63,9 +63,6 @@ def parse_args():
                 client_password = input("mypeugeot password: ")
             web.app.myp.connect(client_email, client_password)
         logger.info(str(web.app.myp.get_vehicles()))
-    if args.offline:
-        logger.info("offline mode")
```
Comment (flobz) [inline server.py:67]: Why did you remove those lines ?

Comment (jlayec) [reply server.py:67]: because except if I made a mistake, they are redundant with lines 52, 53, 54.

**Review by flobz — APPROVED** [review]

**Comment by flobz** [comment]
thanks a lot for your contribution
